core/math/src/main/java/org/arakhne/afc/math/geometry/d3/ai/Path3ai.java

```diff
@@ -1599,7 +1599,7 @@ static void getFarthestPointTo(PathIterator3ai<? extends PathElement3ai> pi, int
 		while (pi.hasNext()) {
 			pe = pi.next();
 
-			boolean foundCandidate = false;
```

Comment:
This change should be discarded because the 3D primitives are currently under intégration (see #47).